.github/workflows/profiles.yml

```diff
@@ -51,6 +51,31 @@ jobs:
       - name: Install
         run: cmake --install build --prefix build/install --config ${{matrix.config}}
 
+  ubuntu-latest:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        config: [Debug, Release]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
+      - name: Install jsonschema
+        run: python3 -m pip install jsonschema
+      - name: Install Dependencies
+        run: sudo apt update && sudo apt install --yes --no-install-recommends libwayland-dev libxrandr-dev
+      - uses: lukka/get-cmake@latest
+        with:
+          cmakeVersion: 3.17.2
```

Comment:
Ensure Vulkan-Profiles builds with exactly `3.17.2`. This ensures stability for system package managers who tend to rely on this functionality.